### PR TITLE
Load fonts asynchronously.

### PR DESCRIPTION
**Pull request by peppy**
Reduces startup time by a magnitude or more (osu! game project).

### CODE REVIEW

File: osu.Framework/Graphics/Drawable.cs [hunk 1]
```diff
@@ -1206,11 +1210,11 @@ public Vector2 ToLocalSpace(Vector2 screenSpacePos)
 
             LoadState = LoadState.Loading;
 
-            await Task.Run(() => Load(game)).ContinueWith(task => game.Schedule(() =>
+            await (loadTask = Task.Run(() => Load(game)).ContinueWith(task => game.Schedule(() =>
```
Comment (Tom94) [inline osu.Framework/Graphics/Drawable.cs]: Can set `loadTask` to null at the end of `Load` (within the lock) such that the object isn't being kept around for the entire lifetime of the Drawable.

File: osu.Framework/Graphics/Drawable.cs [hunk 2]
```diff
@@ -1206,11 +1210,11 @@ public Vector2 ToLocalSpace(Vector2 screenSpacePos)
```
Comment (Tom94) [inline osu.Framework/Graphics/Drawable.cs]: Can you use `nameof(LoadAsync)` in the exception string?

Comment (peppy) [inline osu.Framework/Graphics/Drawable.cs]: which exception do you mean?

Comment (Tom94) [inline osu.Framework/Graphics/Drawable.cs]: Strange... this comment appears on the wrong line suddenly. The one in line 1205

**Pull request merged**